proxy/src/main/java/com/velocitypowered/proxy/connection/forge/modern/ModernForgeConstants.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2018-2023 Velocity Contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package com.velocitypowered.proxy.connection.forge.modern;
+
+/**
+ * Constants for use with Modern Forge systems.
+ */
+public class ModernForgeConstants {
+
+    public static final String MARKER = "\0FORGE";
```

Comment:
Would prefer this name to be marginally more verbose if it's going to be statically imported

Reply:
Changes have been made